shared/lib_battery_powerflow.cpp

```diff
@@ -483,11 +489,8 @@ void BatteryPowerFlow::calculateACConnected()
             P_fuelcell_to_grid_ac = 0;
         }
 
+        // Priliminary batt to grid for DC losses
```

Comment:
this looks like a conjunction of prilliman and preliminary

Reply:
Thanks, fixed.